docs/download-upload-clone/development-plan.md

```diff
@@ -0,0 +1,65 @@
+Each phase assumes that given version of exported feature is fully importable.
+
+# Release 1
+
+* zip file generation
+* zip file storage
+* internal zip file format with versioning set
+* blocking project/scenarios actions during export
+* export project's:
+    * metadata (name etc)
+* export underlying scenarios with:
+    * metadata (name etc)
+
+# Release 2
+
+* project's:
+	* planning area: selected GADM & grid config settings
+	* custom planning area: user-uploaded planning are geometries & grid config settings (first 
+	  shapefile handling)
+	* generated PU with geometries
+
+# Release 3
+
+* project's:
+	* custom grid, settings and generated PU with geometries
+	* custom grid's shapefile (restored, not original)
+
+// note: may happen this release could be done in parallel with r2
+
+# Release 4
+
+* scenario's:
+	* IUCN categories (protected areas) & threshold
+	* features specification (only pre-defined features)
+	* spf/targets
+
+# Release 5
+
+* scenario's:
+    * cost data per PU
+    * lock status
+    * marxan settings
+
+# Release 6
+
+* scenario's:
+  * previous run results
+  * custom protected areas
+
+# Release 7
+
+* scenario's:
+    * custom (uploaded) features
+    * allow to `execute run` when uploading if there were no previous results
```

Comment:
If there were previous results, remember that it might be important for the user to keep those. 
This needs to be checked with the client, tho. But it shouldn't be a blocker to keep going with the plan.

Reply:
Oh, forget. I just realized that is in the previous line. :)